templates/rag-pinecone/README.md

```diff
@@ -33,6 +33,7 @@ And add the following code to your `server.py` file:
 ```python
 from rag_pinecone import chain as rag_pinecone_chain
 
+Be careful with this, in the console, when you create the project add_routes(app, rag_pinecone_chain, path="\rag-pinecone") appears with a backslash, the correct route is "/rag-pinecone"
```

Comment:
should be a comment (prefixed with "# ")

Reply:
@baskaryan I fixed this, lmk if I am good to merge